_content/news/_posts/2018-12-01-liege.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Dames A vs Panthers Liege"
+date: 2018-11-26 20:00:00
+description: Op 1 December spelen onze Dames tegen Panthers Liege, volg het hier live.
+cover: /news/img/jaleesa.jpg
+permalink: /news/2018-12-01-liege/
+---
+
+Op zaterdag 1 December nemen onze Dames A het voor de eerste keer op tegen Panthers Liege. Dit wordt een speciale wedstrijd voor ons, want bij de Panthers speelt onze eigenste Jaleesa Maes. 
+
+## Welkom thuis Jaleesa
+
+Vandaag is Jaleesa opgenomen in de selectie van de Belgian Cats en een vaste waarde in de basis bij Panthers Liege. Maar haar jeugdopleiding heeft deze lummense bij ons genoten. Na 10 jaar heeft ze afscheid genomen en haar professionele carriére verder gezet bij Sint-Katelijne-Waver, Boom, DBC Houthalen en Panthers Liege.
```

Comment:
Tegenwoordig is Jaleesa opgenomen in de selectie van de Belgian Cats en een vaste waarde in de basis bij Liège Panthers. Haar jeugdopleiding heeft deze Lummense echter bij ons genoten. Ze nam na 10 jaar afscheid en zette haar professionele carrière verder bij Sint-Katelijne-Waver, Boom, DBC Houthalen en Liège Panthers.

Reply:
Blijkbaar is ze recentelijk gestopt bij de Cats, zal het proberen verwoorden